central/complianceoperator/checkresults/store/postgres/gen.go

```diff
@@ -0,0 +1,3 @@
+package postgres
+
+//go:generate pg-table-bindings-wrapper --type=storage.ComplianceOperatorCheckResult --table=complianceoperatorcheckresults
```

Comment:
Can you change it to compliance_operator_check_results to be consistent with other stores?